actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -55,12 +63,26 @@ def call(env)
 
       response
     rescue Exception => exception
+      invoke_interceptors(request, exception)
       raise exception unless request.show_exceptions?
       render_exception(request, exception)
     end
 
     private
 
+    def invoke_interceptors(request, exception)
```

Comment:
We could call this method from inside `render_exception`, or should set the `backtrace_cleaner` and `wrapper` variables to be instance variables, so we don't have to create the same thing multiple times :grimacing: